feature-libs/checkout/base/components/checkout-review/checkout-review-shipping/checkout-review-shipping.component.html

```diff
@@ -0,0 +1,88 @@
+<ng-container *ngIf="entries$ | async as entries">
+  <h2 class="cx-review-header">
+    {{ 'checkoutReview.shippingLabel' | cxTranslate }}
+  </h2>
+  <div class="cx-review-summary cx-review-shipping-summary">
+    <ng-container *ngIf="(steps$ | async)?.slice(0, -1) as steps">
+      <div class="cx-review-summary-card-container">
+        <ng-container *ngFor="let step of deliverySteps(steps)">
```

Comment:
This "get all steps, and check each of them" is for express checkout. In your component, no need consider step stuff.